src/libs/OptionsListUtils.js

```diff
@@ -349,7 +350,9 @@ function createOption(logins, personalDetails, report, reportActions = {}, {
                 ? lastMessageText
                 : Str.removeSMSDomain(personalDetail.login);
         }
+        reportName = ReportUtils.getReportName(report, policies);
     } else {
+        reportName = ReportUtils.getDisplayNameForParticipant(logins[0]);
```

Comment:
Just wondering, why use `logins[0]` here instead of `personalDetail.login` like the following 2 lines?